docs/jade-getting-started.md

```diff
@@ -428,31 +406,17 @@ helmfile apply
 helm list --namespace ZZ
 ```
 
-7. Update the following authorized domains within the
-   [Jade Data Repository OAuth2 Client configuration](https://console.cloud.google.com/apis/credentials/oauthclient/970791974390-1581mjhtp2b3jmg4avhor1vabs13b7ur.apps.googleusercontent.com?authuser=0&project=broad-jade-dev):
-
-- Under Authorized JavaScript origins, add `https://jade-ZZ.datarepo-dev.broadinstitute.org`
-- Under Authorized redirect URIs, add `https://jade-ZZ.datarepo-dev.broadinstitute.org/login/google` and
-  `https://jade-ZZ.datarepo-dev.broadinstitute.org/webjars/springfox-swagger-ui/oauth2-redirect.html`
-
-8. Connect to your new dev postgres database instance:
+4. You can access the instance at the following address: `https://jade-ZZ.datarepo-dev.broadinstitute.org`
+5. Connect to your new dev postgres database instance:
    Note that this is a different instance than the local one you will configure in [step 10](#10-install-postgres-11).
    The following command connects to the database via a proxy.
-
 ```
 cd jade-data-repo/ops
 DB=datarepo-ZZ SUFFIX=ZZ ENVIRONMENT=dev ./db-connect.sh
```

Comment:
It looks like this wasn't updated as part of https://github.com/DataBiosphere/jade-data-repo/pull/1731

Actually looking at this more closely I'm not sure how to invoke the script for this case. @fboulnois can you provide the new command line for this operation?

Reply:
That functionality is no longer supported in the `db-connect.sh` script. This decision was based on feedback as a part of https://github.com/DataBiosphere/jade-data-repo/pull/1731#discussion_r1672265259

Reply:
After looking more into this, given that personal deployments will mainly be used for testing helm charts, I don't expect for there to be extensive postgres querying. Users can access the database in the Gcloud UI for some one-off queries.